packages/sitecore-jss-nextjs/src/index.ts

```diff
@@ -38,6 +38,19 @@ export {
   ComponentFields,
   ComponentParams,
   isServer,
+  LayoutPersonalizationService,
+  PersonalizationResult,
+  PersonalizationLoadResult,
+  LayoutFragmentData,
+  LayoutFragmentService,
+  GraphQLLayoutFragmentService,
+  GraphQLLayoutFragmentServiceConfig,
```

Comment:
I do not see RestLayoutFragmentService.
Does it make sense to simplify the names?
E.g. LayoutFragmentService, LayoutFragmentServiceConfig